### ISSUE TITLE
Add donations info

### ISSUE THREAD

**Issue by paulhkz** [m-ou-se/rust-atomics-and-locks#68]
### Where the issue appears

[Startpage](https://marabos.nl/atomics/)
[End](https://marabos.nl/atomics/inspiration.html)

### Description of the issue

Since there are probably many people (like me) who do not necessary want to spent 50 bucks or so for the printed book and have another agency in-between (like Amazon), I would love to rather donate to you directly.

So it would be really nice to add some sort of one-time donations link or information, like either bank-account or crypto (BTC, MON, LTC, whatever).

I also haven't found generally much info how to sponsor you (except for GitHub sponsors) so I cannot make a pull request.
Even if you might say that you don't need the money, I feel bad reading a very good book which covers those very interesting topics in quite some detail *without* paying anything.
